src/hyp3_sdk/hyp3.py

```diff
@@ -487,10 +488,20 @@ def my_info(self) -> dict:
         _raise_for_hyp3_status(response)
         return response.json()
 
-    def check_quota(self) -> Optional[int]:
+    def check_credits(self) -> Optional[float]:
```

Comment:
Hmm, it looks like we only specify "number" in HyP3's OpenAPI spec, which could be a float or an int. I wonder if we should actually make this specify float or int, or always coarse to float. Right now, I suppose they will actually come as ints unless we make a job cost a fractional amount...